django/core/mail/message.py

```diff
@@ -280,9 +280,15 @@ def message(self):
         if "message-id" not in header_names:
             # Use cached DNS_NAME for performance
             msg["Message-ID"] = make_msgid(domain=DNS_NAME)
+
+        if "Auto-Submitted" not in self.extra_headers:
+            # Default to adding the Auto-Submitted : auto-generated header
```

Comment:
There should not be a space after "Auth-Submitted" (also in other places).

Reply:
Actually I'd ask this comment be removed pls. We have a policy of not commenting the "what", focusing on commenting the "why" 👍